test/integration/ConversationCursor.spec.ts

```diff
@@ -29,7 +29,10 @@ describe('Integration - Conversation cursor', () => {
         // Get cursor on specific message
         const cursor = await receiver.getCursorOnMessage(conversationId, messageId, { initialSize: 3 })
 
-        if (!cursor) return
+        if (!cursor) {
+            expect(cursor).to.be.not.empty
+            return
+        }
```

Comment:
I think this should be sth like this:
```suggestion
        expect(cursor).to.be.not.empty
```

Reply:
Yes, I know. I left it that way because otherwise, I would have to add the `!` operator ([commit](https://github.com/decentraland/matrix-client/pull/116/commits/83d3f537ab8ed8eccf22d073d11847ef047f5748)) and I don't know what is considered more idiomatic.

Reply:
Got it! Since this is a test and a well known setup, I think the best is the bang expression, but I would suggest using it here to avoid spreading it downwards.
```typescript
        const cursor = await receiver.getCursorOnLastMessage(conversationId, { initialSize: 20 })!
```

Reply:
Alright! Thx 🙌🏼